### ISSUE TITLE
Adding more images to the database

### ISSUE THREAD

**Issue by 1Shubham7** [1Shubham7/Wife-Predictor-v2#4]
Guys you can find more images online and give me the link. We can expand the number of images in the website

**Comment by Agniv25** [comment]
I want to work on this. Please assign this to me

**Comment by 1Shubham7** [comment]
would love your participation @Agniv25 . Just make sure one thing - after adding images collect all links of images and copy on comments so that I can verify those images. Images should be funny but not disrepectful.

**Comment by theQuery** [comment]
I'd like to be assigned to this task as well, but would I be correct in assuming you also want us to add a name, description, and etcetera?

**Comment by 1Shubham7** [comment]
yup, I want to and trust you. Please checkout the other images, you will find a pattern. We want to make this website a fun project, so please keep that in mind.

**Comment by Gokilp** [comment]
please assign to me #gssoc24

**Comment by Coder-Kartikey** [comment]
Please Assign this to me under GSSOC24. I will also add some random name, address, age and a funny tagline for image.

**Comment by Ashut0shGit** [comment]
Hey @1Shubham7 can I please still contribute to this under GSSOC'24 ? I will ensure that the image database is diverse and inclusive, representing women from various ethnicities, cultures, ages, and backgrounds making the website more creative and fun. 
Looking forward to your response !!
